sdk/batch/azure-resourcemanager-batch/src/test/java/com/azure/resourcemanager/batch/BatchTests.java

```diff
@@ -276,8 +278,11 @@ public void testCRUDBatchPool() {
                 .withDisplayName(poolDisplayName)
                 .withDeploymentConfiguration(
                     new DeploymentConfiguration()
-                        .withCloudServiceConfiguration(
-                            new CloudServiceConfiguration().withOsFamily("4")))
+                        .withVirtualMachineConfiguration(
+                            new VirtualMachineConfiguration()
+                                .withImageReference(new ImageReference().withPublisher("Canonical")
+                                    .withOffer("UbuntuServer").withSku("18.04-LTS").withVersion("latest"))
+                                .withNodeAgentSkuId("batch.node.ubuntu 18.04")))
```

Comment:
com.azure.core.management.exception.ManagementException: Status code 400, "{"error":{"code":"CloudServiceConfigurationRetired","message":"CloudServiceConfiguration pools are retired as of 2/29/2024, and new pools of this type cannot be added\nRequestId:80525c33-84a1-4685-b3c8-ab2a03566f10\nTime:2024-03-27T07:04:01.2339901Z","target":"BatchAccount"}}"

`CloudServiceConfiguration` is not supported anymore.